frontend/app/routes/$lang/_protected/access-to-governmental-benefits/edit.tsx

```diff
@@ -336,12 +336,22 @@ export default function AccessToGovernmentalsBenefitsEdit() {
           />
         </fieldset>
         <div className="flex flex-row-reverse flex-wrap items-center justify-end gap-3">
-          <ButtonLink id="back-button" routeId="$lang/_protected/access-to-governmental-benefits/view" params={params} disabled={isSubmitting}>
+          <ButtonLink
+            id="back-button"
+            routeId="$lang/_protected/access-to-governmental-benefits/view"
+            params={params}
+            disabled={isSubmitting}
+            data-gc-analytics-customclick="ESDC-EDSC:CDCP Online Application Form - Access to Governmental Benefits:Back - Edit click"
```

Comment:
Since this page isn't part of the CDCP online application form,

```suggestion
            data-gc-analytics-customclick="ESDC-EDSC:CDCP Personal Information:Back - Access to other federal, provincial or territorial dental benefits click"
```

So the format will be something like:
`"ESDC-EDSC:CDCP Personal Information:[Button text] - [Current page title] click"`

I am assuming these `/access-to-governmental-benefits` pages will eventually be accessible through the Personal Information section.

(same comment can be applied to other `data-gc-analytics-customclick` values)

Reply:
Makes sense. Implemented in commit 871b8e7da4e7bb6cfed04f2a0d20b0705e984bb8.

Reply:
We should replace "Edit" with the actual page title. For `/access-to-governmental-benefits/edit`, it would be "Access to other federal, provincial or territorial dental benefits". 

No need to have "Edit", "View", or "Update" as these should be replaced by the page title themselves.

We can remove the page title from the second set of text delimited by the colon, so that second set is "CDCP Personal Information".

For this particular button, the value would be:
```
"ESDC-EDSC:CDCP Personal Information:Back - Access to other federal, provincial or territorial dental benefits click"
```

Reply:
Ah, thanks for the clarification.

Implemented in commit ab8760d0eac3b9b6686a87574f614ae0084d89ca.